satellite/gracefulexit/db.go

```diff
@@ -0,0 +1,59 @@
+// Copyright (C) 2019 Storj Labs, Inc.
+// See LICENSE for copying information.
+
+package gracefulexit
+
+import (
+	"context"
+	"time"
+
+	"storj.io/storj/pkg/storj"
+)
+
+// Progress represents the persisted graceful exit progress record
+type Progress struct {
+	NodeID           storj.NodeID
+	BytesTransferred int64
+	UpdatedAt        time.Time
+}
+
+// TransferQueueItem represents the persisted graceful exit queue record
+type TransferQueueItem struct {
+	NodeID          storj.NodeID
+	Path            []byte
+	PieceNum        int
+	DurabilityRatio float64
+	QueuedAt        time.Time
+	RequestedAt     time.Time
+	LastFailedAt    time.Time
+	LastFailedCode  int
+	FailedCount     int
+	FinishedAt      time.Time
```

Comment:
I'm not sure whether there's value in keeping the finished items in the queue.

Reply:
Added delete finished